### ISSUE TITLE
Modus Alert Component - Keyboard navigation - Close icon is not accessible

### ISSUE THREAD

**Issue by HarshithaNous** [trimble-oss/modus-web-components#1541]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-web-components/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-web-components/blob/main/CONTRIBUTING.md)

### Describe the issue

Modus Alert Component - Keyboard navigation - Close icon is not accessible

**Steps to Reproduce:**
1. Access Modus web components storybook page
2. Navigate to Alert Default component
3. On Canvas tab, select controls section
4. On Dismissible, select the control as True
5. Close icon should be displayed on default alert component
6. Hit on tab key and verify if the close icon is focused

**Expected Result:** Close icon should be accessible on Keyboard navigation

**Actual Result:** Close icon is not accessible on Keyboard navigation

Please refer the below screenshot:
![image](https://github.com/trimble-oss/modus-web-components/assets/129380495/9717cf6f-8cbe-4fcb-a548-737cd4944220)

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Firefox, Microsoft Edge

### What is the issue regarding ?

@trimble-oss/modus-web-components

### Priority

Medium

### What product/project are you using Modus Components for ?

QA

### What is your team/division name ?

QA

### Are you willing to contribute ?

None

**Comment by msankaran0712** [comment]
Sprint 72

**Comment by coliff** [comment]
This is fixed now.
Test: https://modus-web-components.trimble.com/?path=/story/components-alert--default&args=dismissible:true
